hera_mc/weather.py

```diff
@@ -0,0 +1,128 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Handling quality metrics of HERA data.
+
+"""
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.time import Time
+from math import floor
+from sqlalchemy import Column, BigInteger, Float, String
+
+import tornado.gen
+
+from . import MCDeclarativeBase
+
+weather_sensor_dict = {'wind_speed': {'sensor_name': 'anc_wind_wind_speed', 'units': 'm/s'},
+                       'wind_direction': {'sensor_name': 'anc_wind_wind_direction', 'units': 'degrees'},
+                       'temperature': {'sensor_name': 'anc_weather_temperature', 'units': 'deg. Celcius'}}
```

Comment:
Spelling mistake (perhaps theirs, though?)

Reply:
no, it was mine, embarrassing as it is